### DISCUSSION TITLE
Import Value Mappings from CSV

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64452] [Dashboards, Panels & Visualizations]
When we have many value mappings for a panel, entering them all manually is pretty tedious. I am looking for a way to have Grafana do this for me automatically from comma-separated values that I paste in or import from a file. Ex:
```
0,Idle
1,Hold
2,Rate Damp
3,Rate Damp Non Zero
4,Sun Acquisition 1
5,Sun Acquisition 2
6,Inertial Point
7,Sun Track Occ
8,Sun Track Nom
9,Nadir Track
10,Velocity Track
11,Velocity Track 2
12,Velocity Track Nadir Clock
13,Ground Station Point
14,New Tracking Mode
15,Mag Field Track
16,Target Track
17,Eigen Slew
18,Rendez Slew
```
A bonus would be to support hex values. Ex:
```
0x00, Idle
0x01, Hold
0x02, Rate Damp
0x03, Rate Damp Non Zero
0x04, Sun Acquisition 1
0x05, Sun Acquisition 2
0x06, Inertial Point
0x07, Sun Track Occ
0x08, Sun Track Nom
0x09, Nadir Track
0x0A, Velocity Track
0x0B, Velocity Track MET
0x0C, Velocity Track Nadir Clock
0x0D, Ground Station Point
0x0E, New Tracking Mode
0x0F, Mag Field Track
0x10, Target Track
0x11, Eigen Slew
0x12, Rendez Slew
```
As I write this, I'm thinking a workaround might be to edit the JSON for the dashboard directly and then reimport the dashboard.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82791. Feel free to continue the discussion around this there. Thank you!
